bat/BLDJAR.BAT

```diff
@@ -12,10 +12,18 @@ set /A z_NestLevel=%z_NestLevel%+1
 rem ----- Lvl(%z_NestLevel%) Start %0 %1 %2 %3 %4 %5 %6 %7 %8 %9
 
 pushd %~dps0..
-if not exist ".\jar\" mkdir jar
-
 rem both the compiler javac.exe and jar builder jar.exe must be on sdk path
 rem note you can use current z390.jar with compatible java runtime with sdk
+
+rem first generate javadoc
+if not exist ".\docenv\javadoc\" mkdir docenv\javadoc
+cd docenv\javadoc
+call javadoc.exe ..\..\src\*.java
+cd ..\..
```

Comment:
Suggestion. Do the javadoc generation in a separate bat file. Someone may want to run BLDJAR.BAT to re-build the z390.jar file. Why should they also re-build the javadoc?

Reply:
I understand your line of reasoning. However, in my mind, it makes sense to generate the doc whenever the code is compiled. Main reasons being: the doc is generated from the very same source that is being compiled. Generating the doc for every compile ensures the doc is always up to date and in sync with the current jar. And since it takes only a few seconds the overhead is manageable. 
I suggest to put the javadoc logic in a separate procedure, which will be called by default when compiling the jar. I'll add a parm to the BLDJAR.BAT so users have an option to suppress the javadoc step. I'll let it default to not suppress the javadoc since in the github logic after every push we do want to generate the latest and greatest.
Would that be okay with you?